tests/unit/progress/test_tables.py

```diff
@@ -60,3 +62,57 @@ def test_table_progress_encoder_pending_migration_failure(mock_backend, table: T
     ownership.owner_of.assert_called_once()
     table_migration_index.is_migrated.assert_called_with(table.database, table.name)
     grant_progress_encoder.assert_not_called()
+
+
+@pytest.mark.parametrize(
+    "used_table",
+    [
+        UsedTable(
+            catalog_name="catalog",
+            schema_name="schema",
+            table_name="table",
+            source_timestamp=dt.datetime.now(dt.timezone.utc),
+            assessment_start_timestamp=dt.datetime.now(dt.timezone.utc),
+            assessment_end_timestamp=dt.datetime.now(dt.timezone.utc),
+        ),
+    ],
+)
```

Comment:
it makes no sense to have a parametrised testcase with just a single line in a test table, as it'll get more difficult to debug. don't abuse dynamic values and just make it an inline variable in a test. same for `test_used_table_progress_encoder_pending_migration_failure`